shared/video-sdk/get-started/get-started-uikit/project-implementation/web.mdx

```diff
@@ -83,27 +98,27 @@ export default App;
 
 ### Display the user interface
 
-If `videoCall` is `true`, show the `AgoraUIKit` interface. If it is `false` show the _Join_ button. To do this, add the provided code sample after the `callbacks` declaration:
+That's the code for the UIKit, if `videoCall` is `true` you'll redner the `AgoraUIKit` component. If it is `false` we show a _Join_ button.
```

Comment:
render.  


if it is `false`, show the _Join_ button. 

(Never say we, avoid the person or say you. )